.github/workflows/image-master.yaml

```diff
@@ -52,329 +30,110 @@ jobs:
       pull-requests: read
       repository-projects: read
       statuses: read
-    secrets: inherit
     with:
-      flavor: ${{ matrix.flavor }}
-      flavor_release: ${{ matrix.flavorRelease }}
-      family: ${{ matrix.family }}
-      base_image: ${{ matrix.baseImage }}
-      model: ${{ matrix.model }}
+      base_image: ${{ matrix.base_image }}
       variant: ${{ matrix.variant }}
-      arch: ${{ matrix.arch }}
+      model: "generic"
+      arch: "amd64"
+      event_type: ${{ github.event_name }}
     strategy:
       fail-fast: false
-      matrix: ${{fromJson(needs.get-core-matrix.outputs.matrix)}}
-  install:
-    secrets: inherit
-    uses: ./.github/workflows/reusable-install-test.yaml
-    with:
-      flavor: ${{ matrix.flavor }}
-      flavor_release: ${{ matrix.flavorRelease }}
-      variant: "core"
-    needs:
-      - core
-    strategy:
-      fail-fast: true
       matrix:
-        include:
-          - flavor: opensuse
-            flavorRelease: leap-15.6
-            secureboot: false
-  install-target:
+        variant: ["core", "standard"]
```

Comment:
I think this used to be only core before, just mentioning in case adding standard wasn't intentional

Reply:
on master it run standard as well, no idea why. I would change this to be PR tests stuff (both core and standard), you cant merge with an outdated branch, and make master only build the artifacts of all flavors for testing the build part, but the full test suite I would do PR only. Once its merged we tend to forget about it.